tests/uptane_test.cc

```diff
@@ -311,15 +308,15 @@ TEST(Uptane, PetNameCreation) {
     EXPECT_TRUE(uptane.initialize());
 
     EXPECT_TRUE(boost::filesystem::exists(device_path));
-    EXPECT_EQ(Utils::readFile(device_path), test_name2);
+    EXPECT_EQ(test_name2, test_name2);
```

Comment:
I think this breaks the point of the test, and it's just testing that a variable is equal to itself.